### PR TITLE
Addon-docs: Increase docs content wrapper max-width to 1000px

### PR DESCRIPTION
**Pull request by patricklafrance**
Issue: https://github.com/storybookjs/storybook/issues/8947

## What I did

- Increased the docs content wrapper max-width to 1000px
- Added new test cases in the cra-ts-kitchen-sink for custom prop type

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes
- Does this need a new example in the kitchen sink apps? No
- Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: examples/cra-ts-kitchen-sink/src/stories/docgen-tests/types/prop-types.js [hunk 1]
```diff
@@ -140,6 +149,9 @@ PropTypesProps.propTypes = {
   obj: PropTypes.object, // eslint-disable-line react/forbid-prop-types
   symbol: PropTypes.symbol,
   node: PropTypes.node,
+  useCustomPropType: customPropType,
```
Comment (shilman) [inline examples/cra-ts-kitchen-sink/src/stories/docgen-tests/types/prop-types.js]: i think you accidentally committed more than you meant?

**Review by shilman — CHANGES_REQUESTED** [review]
What's this moment business? i think this should be a one line PR?

File: examples/cra-ts-kitchen-sink/src/stories/docgen-tests/types/prop-types.js (hunk 1 above)
Comment (patricklafrance) [reply examples/cra-ts-kitchen-sink/src/stories/docgen-tests/types/prop-types.js]: No, well I added a few more test cases to the kitche sink

Comment (patricklafrance) [reply examples/cra-ts-kitchen-sink/src/stories/docgen-tests/types/prop-types.js]: I can remote it

Comment (shilman) [reply examples/cra-ts-kitchen-sink/src/stories/docgen-tests/types/prop-types.js]: Oh sorry I misunderstood the code. Nevermind!!

**Pull request merged**